From 5b42db72a88542b4b13d1cdf0b86dd7a8b51c6a1 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Tue, 31 Mar 2026 20:06:43 +0800
Subject: [PATCH 7/7] AOSCOS: drop IBM1390/1399 for now

IBM1390 and 1399 are affected by CVE-2026-4046.  I don't think any AOSC
OS users really need them, so drop them.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 iconvdata/Makefile                 |  2 +-
 iconvdata/gconv-modules-extra.conf | 14 --------------
 2 files changed, 1 insertion(+), 15 deletions(-)

diff --git a/iconvdata/Makefile b/iconvdata/Makefile
index 5a2abeea24..2b173a09e4 100644
--- a/iconvdata/Makefile
+++ b/iconvdata/Makefile
@@ -60,7 +60,7 @@ modules	:= ISO8859-1 ISO8859-2 ISO8859-3 ISO8859-4 ISO8859-5		 \
 	   IBM1142 IBM1143 IBM1144 IBM1145 IBM1146 IBM1147 IBM1148	 \
 	   IBM1149 IBM1166 IBM1167 IBM4517 IBM4899 IBM4909 IBM4971	 \
 	   IBM5347 IBM9030 IBM9066 IBM9448 IBM12712 IBM16804             \
-	   IBM1364 IBM1371 IBM1388 IBM1390 IBM1399 ISO_11548-1 MIK BRF	 \
+	   IBM1364 IBM1371 IBM1388 ISO_11548-1 MIK BRF	 \
 	   MAC-CENTRALEUROPE KOI8-RU ISO8859-9E				 \
 	   CP770 CP771 CP772 CP773 CP774
 
diff --git a/iconvdata/gconv-modules-extra.conf b/iconvdata/gconv-modules-extra.conf
index edd8c87cda..5ee6fdcc3b 100644
--- a/iconvdata/gconv-modules-extra.conf
+++ b/iconvdata/gconv-modules-extra.conf
@@ -1818,20 +1818,6 @@ alias	CSIBM1388//		IBM1388//
 module	IBM1388//		INTERNAL		IBM1388		1
 module	INTERNAL		IBM1388//		IBM1388		1
 
-#	from			to			module		cost
-alias	IBM-1390//		IBM1390//
-alias	CP1390//		IBM1390//
-alias	CSIBM1390//		IBM1390//
-module	IBM1390//		INTERNAL		IBM1390		1
-module	INTERNAL		IBM1390//		IBM1390		1
-
-#	from			to			module		cost
-alias	IBM-1399//		IBM1399//
-alias	CP1399//		IBM1399//
-alias	CSIBM1399//		IBM1399//
-module	IBM1399//		INTERNAL		IBM1399		1
-module	INTERNAL		IBM1399//		IBM1399		1
-
 #	from			to			module		cost
 alias	ISO/TR_11548-1/		ISO_11548-1//
 alias	ISO11548-1//		ISO_11548-1//
-- 
2.53.0

